notebooks/01 - pandas basics.ipynb

```diff
@@ -4,35 +4,51 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# pandas basics\n",
+    "# Pandas Basics\n",
```

Comment:
The name "pandas" is usually lowercased, also at the beginning of the sentence (see e.g. https://pandas.pydata.org/)